packages/client/package.json

```diff
@@ -85,7 +86,7 @@
     "@types/jest": "28.1.6",
     "@types/js-levenshtein": "1.1.1",
     "@types/mssql": "8.0.3",
-    "@types/node": "12.20.55",
+    "@types/node": "14.17.34",
```

Comment:
I wanted to have `fs/promises` definition. What do we usually pin `@types/node` to? Lowest supported version or latest supported version?

Reply:
Lowest supported version, so `^14.17.0` should be fine

Reply:
AFAIK, patch version in `@types/node` does not correspond to node.js version (node 14.17.34 does not even exist) and denotes type-specific fixes, so I guess it would be ok to leave `14.17.34`